app/api/models/region_validation.py

```diff
@@ -26,65 +25,74 @@ class RegionValidation(BaseModel):
     location_input: str
     genome_uuid: str = None
     name: str = Field(alias="name", default="")
-    start: int = Field(alias="start", default=0)
-    end: int = Field(alias="end", default=0)
+    start: int = Field(alias="start", default=1)
+    end: Optional[int] = Field(alias="end", default=None)
     _region_code: str = None
     _is_valid: [bool, bool, bool] = [False, False, False]
-    _region_name_em : str = None
-    _start_em : str = None
-    _end_em : str = None
+    _region_name_error: str = None
+    _start_error: str = None
+    _end_error: str = None
 
     @root_validator(pre=True)
     def set_region_parameters(cls, values):
         parsed_region = cls.parse_location_input(values.get("location_input"))
         values["name"], values["start"], values["end"] = parsed_region
         return values
 
-    def parse_location_input(rgn_input):
-        rn, start, end = "", 0, 0
-        logger.debug(rgn_input)
+    @staticmethod
+    def parse_location_input(region_input):
+        region, start, end = "", 1, None
         try:
-            rn_coords = rgn_input.split(":")
-            if len(rn_coords) == 2:
-                rn = rn_coords[0]
-                start, end = rn_coords[1].split("-")
-                start = start.replace(",","")
-                end = end.replace(",","")
+            region_coordinates = region_input.split(":")
+            region = region_coordinates[0]
+            start, end = region_coordinates[1].split("-")
+            start = start.replace(",", "")
+            end = end.replace(",", "")
+        except IndexError:
+            pass
```

Comment:
logger.debug() ?

Reply:
It's an expected error that I want to purposely pass . we don't need a logger for that, I kept the one in the next line for unexpected errors.